.github/ISSUE_TEMPLATE/01-bug-report-plugin.yaml

```
@@ -1,4 +1,4 @@
-name: '🐛 Report a Bug With a Rule'
+name: '🐛 Rule Bug'
 description: 'Report a bug you encountered with a lint rule'
```

[Question] I'd also looked at simplifying descriptions, such as this one:

```suggestion
description: 'An issue with a single lint rule'
```

Was it an intentional decision to not make any of those changes here?

---

I kept the descriptions under the assumption that if a person bothers to look at it, they might be confused about the title or look for extra clarification since it's easy to skip the descriptions when looking for the specific issue category you want to file 👍 

However, I can absolutely see why one may want to shorten the descriptions, so we can tackle this in another PR.